docs/crumbs.html

```diff
@@ -23,6 +23,7 @@
 <span class="dropdown">
   <span class="dropbtn">**Tutorials**</span>
   <span class="dropdown-content">
+    [**`strlen`**](tutorial/strlen.html)
```

Comment:
There is no "basic loop tutorial" at the moment.
Generally speaking anything is a bit more complicated then this strlen tbh.
I also think this is a good starting point because:
a) it show cases that you can use the library to do very real things that everyone is familiar with
b) it show cases a good number of library concepts.